### ISSUE TITLE
kelvin-connect the TEC divider to ground near the reference

### ISSUE THREAD

**Issue by hartytp** [sinara-hw/Thermostat#58]
to minimize sensitivity to ground loops...

**Comment by hartytp** [comment]
Let's also use star-routing on the Vref+ TEC lines to avoid cross-talk between the two TEC lines. Basically, have stars on both TEC+ and TEC- that go to the ADC REF+ and REF- lines.

**Comment by gkasprow** [comment]
do you mean this ground?
![obraz](https://user-images.githubusercontent.com/4325054/69287877-0dd3ca00-0bf8-11ea-9847-6761c7d37824.png)

**Comment by gkasprow** [comment]
TEC+ and TEC- are routed to the diff amplifiers. And the divider ground should indeed be connected as close to CPU as possible.

**Comment by gkasprow** [comment]
OK, so I implemented star connection for both ref and divider GND
![obraz](https://user-images.githubusercontent.com/4325054/69288548-38268700-0bfa-11ea-8f24-6da38b1f6883.png)
![obraz](https://user-images.githubusercontent.com/4325054/69288564-470d3980-0bfa-11ea-9658-acd6659c2623.png)

**Comment by hartytp** [comment]
@gkasprow sorry! I meant that we should kelvin-connect the thermistor v_ref+ and v_ref- to near the ADC. Not the TEC driver.

**Comment by hartytp** [comment]
i.e. what I want to do here is (a) fix the ground loop that caused the ADC offset to vary with the TEC current (b) minimize the coupling between the two ADC channels.

**Comment by gkasprow** [comment]
In such case, one has to make sure that return current does not pass via sensitive analog circuits.
It should be enough to add a plane cutout that will make the current loop much smaller.

![obraz](https://user-images.githubusercontent.com/4325054/69327487-ea3b6e80-0c4d-11ea-9f3a-38eed45fd4d6.png)

I also added Kelvin connections for thermistors. I used net ties to make sure the vref- won't be shorted to GND by coincidence.
